documentation/azure-gcp-workload-identity-federation.md

```diff
@@ -0,0 +1,23 @@
+# Azure GCP Workload Identity Federation
+
+This allows Azure AKS client to access GCP resources without the use of plain text API Keys, that are considered a security risk.
+
+Workload Identity Federation requires configuration on both Azure and GCP.
+
+## GCP Configuration
```

Comment:
How do you authenticate?
Which level of permission do you need?
How to get a GCP account?

Reply:
Now added.